saleor/core/transactions.py

```diff
@@ -8,11 +9,16 @@ def transaction_with_commit_on_errors():
     """Perform transaction and raise an error in any occurred."""
     error = None
     with transaction.atomic():
-        try:
-            yield
-        except DatabaseError:
-            raise
-        except Exception as e:
-            error = e
+        with opentracing.global_tracer().start_active_span(
+            "transaction_with_commit_on_errors"
```

Comment:
Does this need a separate span name? Tracing will show that this was not aborted if an exception is thrown.

Reply:
Good point, changed it to just use `traced_atomic_transaction`.